src/lib/deploy.ts

```diff
@@ -146,7 +146,14 @@ export class DeploymentsManager {
       retry.delay,
     )
 
-    const errorMessage = Output?.data?.output || error
+    let errorMessage = null
+
+    if (Output && Output.data && Output.data.output) {
+      errorMessage = Output.data.output
+    }
+    else if (error && Object.keys(error).length > 0) {
```

Comment:
I have seen `Error` to be string most of the times if present and it is absent while no error exists. But now we know it can be `Object` too. The code works but lets convert error of type `Object` to string while assigning it to `errorMessage`.

Reply:
sorry for the long reply/fix
let me know if this works

Reply:
I think it should work. Seems you deleted `pnpm-lock.yaml`

Reply:
fixed, can't remember why I deleted, probably was a misclick on my editor